CONTRIBUTING.md

```diff
@@ -21,6 +21,14 @@ phpize > /dev/null && \
 make clean > /dev/null && make all > /dev/null && make install
 ```
 
+To verify that the installation was successful, run the following command which indicates the version of the installed
+extension. It must end with `-dev` suffix.
+
+```
+php -r "echo (new ReflectionExtension('mongodb'))->getVersion();"
```

Comment:
It might also be sensible to point out `php --ri mongodb`, which will print the MongoDB extension portion of phpinfo, including the build configuration (e.g. libmongoc/libmongocrypt versions being used).

Reply:
That's much better. 
```
php --ri mongodb

mongodb

MongoDB support => enabled
MongoDB extension version => 1.16.0-dev
MongoDB extension stability => devel
libbson bundled version => 1.24.0-20230504+git89717a3beb
libmongoc bundled version => 1.24.0-20230504+git89717a3beb
libmongoc SSL => enabled
libmongoc SSL library => Secure Transport
libmongoc crypto => enabled
libmongoc crypto library => Common Crypto
libmongoc crypto system profile => disabled
libmongoc SASL => enabled
libmongoc ICU => disabled
libmongoc compression => enabled
libmongoc compression snappy => disabled
libmongoc compression zlib => enabled
libmongoc compression zstd => enabled
libmongocrypt bundled version => 1.8.0-alpha0
libmongocrypt crypto => enabled
libmongocrypt crypto library => Common Crypto
crypt_shared library version => unknown

Directive => Local Value => Master Value
mongodb.debug => no value => no value
```